tests/integration/cli/classification/test_classification.py

```diff
@@ -81,6 +82,7 @@ def tmp_dir_path():
         yield Path(tmp_dir)
 
 
+MULTI_GPU_UNAVAILABLE = torch.cuda.device_count() <= 1
```

Comment:
How about `MULTI_GPU_SUPPORTED` or `MULTI_GPU_SUPPORT`

Reply:
It means multi GPU isn't available. MULTI_GPU_SUPPORTED = True, when multi GPU isn't available is weird